metals-docs/src/main/scala/docs/UserConfigurationModifier.scala

```diff
@@ -15,20 +15,6 @@ class UserConfigurationModifier extends StringModifier {
       reporter: Reporter
   ): String = {
     info match {
-      case "system-property" =>
```

Comment:
Note that the system properties that are still valid are still in the docs, they were hardcoded there. So this will only be removing the `UserConfiguration` generated system properties which are for almost all the cases not valid.